keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -259,66 +239,76 @@ the API:
   forwarding requests or persisting data) cannot be undone, but no future
   actions should be allowed.
 
-The examples below include information about what happens when the ReferenceGrant
-is removed as data points.
-
 ### Example Usage
 
-#### Gateway API Gateway Referencing Secret
+#### Gateway API Cross-Namespace Secret Reference
 
-In this example (from the Gateway API docs), we have a Gateway in the
-`gateway-api-example-ns1` namespace, referencing a Secret in the
-`gateway-api-example-ns2` namespace. The following ReferenceGrant allows this:
+Authors of APIs can bundle ClusterReferencePatterns with their API definitions
+for any references they expect to be common. For example, Gateway API could
+bundle the following ClusterReferencePattern to describe how Gateways reference
+`kubernetes.io/tls` Secrets. Note that this also allows API authors to define a
+`SameNamespace` `baselineGrant` that would automatically authorize all
+references following this pattern without crossing namespace boundaries.
```

Comment:
(for clarity)
```suggestion
`baselineGrant: SameNamespace`; setting the `baselineGrant` of a ClusterReferencePattern
to `SameNamespace` would automatically authorize all references that match the
ClusterReferencePattern and that do not cross namespace boundaries.
```

(this is more like how we'd write it in the tech docs; I'm recommending it here because I actually thought
there might be a BaselineGrant API that I couldn't see).